src/scripts/services/dimSettingsService.factory.js

```diff
@@ -38,10 +38,10 @@ function SettingsService($rootScope, SyncService, $window, $translate) {
     showNewItems: false,
     // Show animation of new item overlay on new items
     showNewAnimation: true,
-    // Show elemental damage icons
-    showElements: false,
+    // Show reviews damage icons
+    showReviews: true,
     // Can we post identifying information to DTR?
-    allowIdPostToDtr: false,
+    allowIdPostToDtr: true,
```

Comment:
I think the plan is on final release we're going to have a blocking overlay/modal that has it checked by default with the option to uncheck from that modal